src/Pmi/WorkQueue/WorkQueue.php

```diff
@@ -643,4 +646,14 @@ public function getRetentionType($value)
                 return self::HTML_DANGER . ' (Not Retained)';
         }
     }
+
+    public static function getSalivaCollectionMethod($value)
+    {
+        if ($value === 'MAIL_KIT') {
+            return ' Mail Kit';
+        } elseif ($value === 'ON_SITE') {
+            return ' On Site';
```

Comment:
Do we need the leading space with these for display purposes?

Reply:
Yes, I think we added leading space in other places too just to keep the text underneath with other texts.